docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,272 @@
+:::tip
+Snowtrace provides a [unit converter](https://snowtrace.io/unitconverter) between different units
+:::
+
+Run the export script:
+
+```sh
+$ ts-node examples/avm/buildExportTx-cchain-avax.ts
+```
+
+This returns:
+
+```sh
+Success! TXID: Rgg2412kaczRYC3taasvG6bYoqG7tBQG6WfacNdumKDKsVWpF
+```
+
+#### Import the Avax Token From X-Chain to C-Chain
+
+Select the [**`examples/evm`**](https://github.com/ava-labs/avalanchejs/tree/master/examples/evm) folder to view the AvalancheJS C-Chain examples. To import AVAX to the C-Chain from the X-Chain, select [`evm/buildImportTx-xchain.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildImportTx-xchain.ts)
+
+Copy the [network setting from above](#network-setting) into `evm/buildImportTx-xchain.ts`.
+
+Navigate to this part of the code and ensure that the `cHexAddress`(_Your C-Chain wallet address_) and `private key` are correct:
+
+```ts
+const cHexAddress: string = "<YOUR-CCHAIN-WALLET-ADDRESS-HERE>";
+const privKey: string = "<YOUR-PRIVATE-KEY-HERE>";
+```
+
+Run the import script:
+
+```sh
+$ ts-node examples/evm/buildImportTx-xchain.ts
+```
+
+This returns:
+
+```sh
+Success! TXID: r2yYqcnCJcdeV5gddZ8NUoG5ZD3Ef7DxbkiE9xn4RxFcDdMd1
+```
+
+That's it! You've transferred AVAX from the X-Chain to C-Chain!
+
+#### Verify the Transaction
+
+You can now pass this txID `r2yYqcnCJcdeV5gddZ8NUoG5ZD3Ef7DxbkiE9xn4RxFcDdMd1` into [examples/avm/getTx.ts](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/getTx.ts), plus other similar network settings, then you can run
+
+```zsh
+$ ts-node examples/avm/getTx.ts
+```
+
+which returns:
+
+```js
+{
+  unsignedTx: {
+    networkID: 1,
+    blockchainID: '2oYMBNV4eNHyqk2fjjV5nVQLDbtmNJzq5s3qs3Lo6ftnC6FByM',
+    outputs: [],
+    inputs: [ [Object] ],
+    memo: '0x',
+    destinationChain: '11111111111111111111111111111111LpoYY',
+    exportedOutputs: [ [Object] ]
+  },
+  credentials: [
+    {
+      fxID: 'spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ',
+      credential: [Object]
+    }
+  ]
+}
+```
+
+### Transfer from the C-Chain to the X-Chain
+
+To return the AVAX back to the X-Chain, you need to do the transfer in the opposite direction:
+
+- Select the [**`examples/evm`**](https://github.com/ava-labs/avalanchejs/tree/master/examples/evm) folder to view the AvalancheJS C-Chain examples. To export AVAX from the X-Chain to the C-Chain, select [`evm/buildExportTx-xchain-avax.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-xchain-avax.ts)
+
+- Make necessary changes as above for private key and network settings.
+
+- You can change the amount of AVAX to send by editing the _BN_ variable: `avaxAmount`. The sample code assigns this as `1e7` or `10000000` (0.01 AVAX)
+
+The fee here will only be for exporting the asset. The import fees will be deducted from the UTXOs present on the Exported Atomic Memory, a memory location where UTXOs stay after getting exported but before being imported.
```

Comment:
Please describe the code for import in a separate paragraph with `examples/avm/buildImportTx-cchain.ts` so it is much clear instead of lumping them together here.

Reply:
Is this sentence `The import fees will be deducted from the UTXOs present on the Exported Atomic Memory, a memory location where UTXOs stay after getting exported but before being imported.` still apply here?

Reply:
Yes it does

It is informational and describes the fee mechanism

Reply:
But this section focuses on `Export the Avax Token From C-Chain to X-Chain`, why is Import fee being brought up? This should be moved to the next section `Import the Avax Token From C-Chain to X-Chain`.

Reply:
We need to pay the import fees while exporting so it does make sense

also this code:

```js
let avaxAmount: BN = new BN(1e7);
let fee: BN = baseFee.div(new BN(1e9));
fee = fee.add(new BN(1e6));
```

exists on the export script we are using

Reply:
Makes more sense in the export from X-Chain section

Moving now